files/origin-components/console-config.yaml

```diff
@@ -0,0 +1,21 @@
+kind: AssetConfig
```

Comment:
This predates this PR, but this isn't the right name for this directory.  And the directories should be versioned by major directory, because even with 3.10 we're going to have to change these config files across API versions.

Probably should be

```
files/components/web-console/3.9/config.yaml
files/components/web-console/3.9/deployment.yaml
files/components/web-console/3.9/rbac.yaml
```

Reply:
Am I missing something obvious?  Why aren't these files in the files dir of their respective roles, i.e., role/openshift-web-console/files/{config,deployment,rbac}.yaml?  

As for versioning, I'd expect the customer to only be consuming these from the RPMS for the specific version they're installing.